testing-2023-05.md

```diff
@@ -0,0 +1,82 @@
+# napari metadata plugin testing
+
+Author: Lucy Obus (@lcobus)
+
+Date: 2023, May
+
+
+## Summary
+
+- CZI Imaging Tech was motivated to help improve the way napari handles layer metadata, with particular interest in essential metadata attributes like axis names and units that can be critical for accurate processing and visualization of biological data.
+- Andy Sweet decided to see how much improvement could be made by contributing a napari plugin that uses and augments the existing napari public API.
+- To support engineering effort, a targeted concept testing study of the prototype was run by @lcobus with support from @chili-chiu to evaluate plugin feature intuitiveness, overall user value, and learn from participants about the tradeoffs between application as a plugin vs contributing to core napari.
+- We spoke with several community members including @JoOkuma @ziw-liu @toloudis , with the goal of hearing from a diversity of voices that include the OME, AICSIO, napari core developer, and iohub communities. We also did internal testing with CZI application scientists prior to external testing.
```

Comment:
```suggestion
- We spoke with several community members including [Jordão Bragantini](https://github.com/JoOkuma), [Ziwen Liu](https://github.com/ziw-liu), and [Daniel Toloudis](https://github.com/toloudis), with the goal of hearing from a diversity of voices that include the OME, AICSIO, napari core developer, and iohub communities. We also did internal testing with CZI application scientists prior to external testing.
```

Reply:
thank you for making the updates to tag folks!